sync-gateway-config.json

```diff
@@ -97,10 +96,12 @@ function(doc, oldDoc){
     access(doc.owner, "task-list." + doc._id + ".users");
   } else if (getType() == "task") {
     /* Write Access */
+    var owner = doc._deleted ? oldDoc.taskList.owner : doc.taskList.owner;
```

Comment:
If a task document comes in that's missing the taskList property, it's going to fail with a 500 Internal Error when it tries to access doc.taskList.owner.  If you don't want to use a top-level property (taskList_owner), you might want to return a more useful error message when taskList is missing.